### PR TITLE
Package gtkwave

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/gtkwave/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (isuruf) [inline recipes/gtkwave/meta.yaml:21]:
```suggestion
    - {{ compiler('cxx') }}
    - pkg-config
```

File: recipes/gtkwave/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+./configure --prefix=$PREFIX --with-tcl=$PREFIX/lib --with-tk=$PREFIX/lib --disable-xz
+make
```
Comment (isuruf) [inline recipes/gtkwave/build.sh:3]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/gtkwave/build.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+make install
```
Comment (isuruf) [inline recipes/gtkwave/build.sh:4]: Is there a make check?

File: recipes/gtkwave/build.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+./configure --prefix=$PREFIX --with-tcl=$PREFIX/lib --with-tk=$PREFIX/lib --disable-xz
```
Comment (isuruf) [inline recipes/gtkwave/build.sh:2]: Why disable xz?

File: recipes/gtkwave/build.sh (hunk 3 above)
Comment (davidbrochart) [reply recipes/gtkwave/build.sh:4]: Yes, and also a `make installcheck`.

File: recipes/gtkwave/build.sh (hunk 4 above)
Comment (davidbrochart) [reply recipes/gtkwave/build.sh:2]: It could not find it, I'm re-enabling it.

**Comment by davidbrochart** [comment]
BTW I'm unsubscribed from this PR and I can't change that, not sure why.

**Comment by davidbrochart** [comment]
Enabling `xz` doesn't seem to be a problem here (it was locally), is it good to be merged @isuruf ?

**Pull request merged**